types/node/test/crypto.ts

```diff
@@ -1179,11 +1180,10 @@ import { promisify } from 'node:util';
     };
 
     cert.checkEmail('test@test.com'); // $ExpectType string | undefined
-    cert.checkEmail('test@test.com', checkOpts); // $ExpectType string | undefined
+    cert.checkEmail('test@test.com', { subject: checkOpts.subject }); // $ExpectType string | undefined
```

Comment:
```suggestion
    cert.checkEmail('test@test.com', { subject: checkOpts.subject }); // $ExpectType 'email'| undefined
```

Reply:
As my other comment, `string` is the valid type here not `'email'`